src/HeaderCell.tsx

```diff
@@ -128,7 +145,7 @@ export default function HeaderCell<R, SR>({
     <div
       role="columnheader"
       aria-colindex={column.idx + 1}
-      aria-sort={sortColumn === column.key ? getAriaSort(sortDirection) : undefined}
+      aria-sort={sortDirection ? getAriaSort(sortDirection) : undefined}
```

Comment:
I think we can do that. The only thing is `aria-sort` should only be applied to 1 column from what I understand. But let's just add it to any column that is sorted. It will be helpful for testing

https://www.w3.org/TR/wai-aria-1.2/#dom-ariamixin-ariasort
> Authors SHOULD only apply this property to table headers or grid headers. If the property is not provided, there is no defined sort order. For each table or grid, authors SHOULD apply aria-sort to only one header at a time.

https://www.w3.org/TR/wai-aria-practices-1.2/#gridAndTableProperties_sort
> It is important to note that ARIA does not provide a way to indicate levels of sort for data sets that have multiple sort keys. Thus, there is limited value to applying aria-sort with a value other than none to more than one column or row.